tools/cbmc/proofs/settings.json

```diff
@@ -0,0 +1,58 @@
+{
+  "variables": {
+    "UNWIND_LIMIT": 1,
+    "FREERTOS": "../../..",
+    "PROOFS": ".",
+    "CBMC_MAX_OBJECT_SIZE": "\"(UINT32_MAX>>CBMC_OBJECTID_BITS)\"",
+    "CBMC_OBJECTID_BITS": 7,
```

Comment:
We should default to a smaller number like 4, since 32 bits gives very little room for offsets.

We should use better names.  How about
CBMC_OBJECT_BITS
CBMC_OBJECT_MAX_SIZE